docs/CACHE.md

```diff
@@ -21,6 +21,7 @@ To disable it, set ttl to 0s.
 To configure redis cache, use the following options:
 ```
 --redis-cache-addr string         The address of a redis server for cacheID to layer sha mapping
+--redis-password string           The password of the Redis server, should match 'requirepass' in redis.conf
```

Comment:
do you mind renaming this to redis-cache-password for consistency?

Reply:
Sure thing, done.

Please let me know if any other naming/code should be changed 👍